src/Microsoft.DotNet.Interactive.ApiCompatibility.Tests/ApiCompatibilityTests.Interactive_api_is_not_changed.approved.txt

```diff
@@ -229,10 +229,12 @@ Microsoft.DotNet.Interactive
     public System.Threading.Tasks.Task<Microsoft.DotNet.Interactive.Connection.ProxyKernel> ConnectProxyKernelOnDefaultConnectorAsync(System.String localName, System.Uri remoteKernelUri, System.String[] aliases = null)
     public System.Void Dispose()
   public class KernelInfo
-    .ctor(System.String localName, System.String[] aliases = null)
+    .ctor(System.String localName, System.String[] aliases = null, System.Boolean isProxy = False, System.Boolean isComposite = False)
```

Comment:
Having two different constructors with slightly different arguments is a little confusing and probably pointless given all the setters on this class. Maybe we should remove both constructors and just have a default constructor?